xds/internal/server/rds_handler.go

```diff
@@ -131,3 +130,33 @@ func (rh *rdsHandler) close() {
 		cancel()
 	}
 }
+
+// rdsWatcher implements the xdsresource.RouteConfigWatcher interface and is
+// passed to the WatchRouteConfig API.
+type rdsWatcher struct {
+	parent *rdsHandler
+	logger *internalgrpclog.PrefixLogger
+	name   string
```

Comment:
Optional: routeName? I don't know what this name refers to if I'm just looking at this type.

Reply:
Done.